addons/payment/views/payment_provider_views.xml

```diff
@@ -48,11 +49,12 @@
                             </div>
                         </button>
                     </div>
-                    <field name="image_128" widget="image" class="oe_avatar"/>
-                    <widget name="web_ribbon" title="Disabled" bg_color="bg-danger" attrs="{'invisible': [('state', '!=', 'disabled')]}"/>
-                    <widget name="web_ribbon" title="Test Mode" bg_color="bg-warning" attrs="{'invisible': [('state', '!=', 'test')]}"/>
+                    <field name="image_128" widget="image" class="oe_avatar"
+                           attrs="{'readonly': [('module_state', '!=', 'installed')]}"/>
+                    <widget name="web_ribbon" title="Disabled" bg_color="bg-danger" attrs="{'invisible': ['|', ('module_state', '!=', 'installed'), ('state', '!=', 'disabled')]}"/>
+                    <widget name="web_ribbon" title="Test Mode" bg_color="bg-warning" attrs="{'invisible': ['|', ('module_state', '!=', 'installed'), ('state', '!=', 'test')]}"/>
                     <div class="oe_title">
-                        <h1><field name="name" placeholder="Name"/></h1>
+                        <h1><field name="name" placeholder="Name" attrs="{'readonly': [('module_state', '!=', 'installed')]}"/></h1>
```

Comment:
I don't think that this change is correct. Users should be able to change the name of providers created manually (they don't have a module so `module_state=False`), and it's not a big issue if they change the name of a provider before its module is installed.
```suggestion
                        <h1><field name="name" placeholder="Name"/></h1>
```